transform/mattermost-analytics/models/marts/sales/dim_customers.sql

```diff
@@ -0,0 +1,8 @@
+select
+    customer_id
+    , contact_first_name
+    , contact_last_name
+    , name
+    , email
+from 
+    dbt_staging.stg_stripe__customers
```

Comment:
```suggestion
    {{ ref('stg_stripe__customers') }}
```

Let's use DBT's reference. This takes care of referencing schema etc.